src/views/SystemProfile/components/TeamAndContract/index.tsx

```diff
@@ -0,0 +1,323 @@
+    contractAwardDate: 'March 19, 2021',
+    popStartDate: 'March 20, 2021',
+    popEndDate: 'March 21, 2021',
+    contractNumber: 'GS1234567890BA-987654321',
+    technologyFunctions: [
+      'Application',
+      'Delivery',
+      'End User',
+      'IT Management',
+      'Platform',
+      'Security & Compliance'
+    ],
+    assetsOrServices: ['External Labor', 'Software']
+  },
+  {
+    vendors: ['SkyNet'],
+    contractAwardDate: 'April 19, 2021',
+    popStartDate: 'April 20, 2021',
+    popEndDate: 'April 21, 2021',
+    contractNumber: 'GS1234567890BA-123456789',
+    technologyFunctions: ['Network', 'Storage'],
+    assetsOrServices: ['Outside Services']
+  }
+];
+
+const contactListData = [
+  {
+    id: '1',
+    role: 'Business Owner',
+    name: 'Geraldine Hobbs',
+    email: 'geraldine.hobbs@cms.hhs.gov',
+    org: 'Web and Emerging Technologies Group (WETG)'
+  },
+  {
+    id: '2',
+    role: 'System Owner',
+    name: 'Bryce Greenwood',
+    email: 'bryce.greenwood@cms.hhs.gov',
+    org: 'Web and Emerging Technologies Group (WETG)'
+  }
+];
+
+const pointOfContactData = {
+  name: 'Greta May Jones',
+  role: 'Contracting Officer’s Representative (COR)',
+  email: 'gmj@gmj.gov'
+};
+
+const SystemTeamAndContract = ({
+  className,
+  children,
+  system
+}: TeamAndContractProps) => {
+  const { t } = useTranslation('systemProfile');
+  const isMobile = useCheckResponsiveScreen('tablet');
+  return (
+    <div id="system-team-and-contract">
+      <GridContainer className="padding-top-2">
+        <Grid row gap>
+          <Grid desktop={{ col: 9 }}>
+            <SectionWrapper borderBottom className="padding-bottom-5">
+              <PageHeading
+                headingLevel="h2"
+                className="margin-top-0 margin-bottom-4"
+              >
+                {t('teamAndContract.header.teamAndContract')}
+              </PageHeading>
+              <GridContainer className="padding-x-0">
+                <Grid row>
+                  <Grid tablet={{ col: true }}>
+                    <DescriptionTerm
+                      term={t('teamAndContract.federalFullTimeEmployees')}
+                    />
+                    <DescriptionDefinition
+                      className="font-body-md line-height-body-3"
+                      definition="5"
+                    />
+                  </Grid>
+                  <Grid tablet={{ col: true }}>
+                    <DescriptionTerm
+                      term={t('teamAndContract.contractorFullTimeEmployees')}
+                    />
+                    <DescriptionDefinition
+                      className="font-body-md line-height-body-3"
+                      definition="6"
+                    />
+                  </Grid>
+                </Grid>
+              </GridContainer>
+            </SectionWrapper>
+            <SectionWrapper borderBottom className="padding-bottom-5">
+              <PageHeading
```

Comment:
The PageHeading component has an auto focus built in that causes the page to jump around.  I had initially thought of modifying the component to have an optional prop to bypass.  In this case, it's best to use standard h tags for these subheaders.  We can revisit in the future if we want to modify the PageHeading or create another component for subheadings.  PageHeading seems to imply there should only be 1 on a page, especially if it's h1